pkg/storaged/block/resize.jsx

```diff
@@ -284,6 +284,12 @@ export function get_resize_info(client, block, to_fit) {
                 grow_needs_unmount: false
             };
             shrink_excuse = _("VDO backing devices can not be made smaller");
+        } else if (client.blocks_swap[block.path]) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19843-20240117-111611-70e14048-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)